app-authz-springboot/pom.xml

```diff
@@ -41,7 +43,12 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
+            <version>${httpclient.version}</version>
```

Comment:
Why are all these dependency changes needed? Does this indicate some breaking changes in our POMs?

Reply:
This artifact is required by the "TestHelper" classes to upload a Realm JSON.

I'm not sure why not all of them were needed in the past. It's probably one of those 2:
* There was some split in `httpcomponents`. I *think* in the past they were in a single artifact. As the time passed by, it was divided into multiple artifacts?
* Maybe it was provided by Spring?

Reply:
TestHelper AFAIK uses the Keycloak admin client, so this may indicate breaking changes to the Keycloak admin client POM.

Reply:
All changes to POMs I would like to know why they are needed. If they are only test things that's better (but none of the changes you have made are test scoped). Do these indicate some breaking changes on our end?